packages/slice-machine/src/legacy/lib/builders/common/Zone/components/ZoneEmptyState/ZoneEmptyState.module.css

```diff
@@ -0,0 +1,12 @@
+.root {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 16px;
+  background-color: var(--grey3);
+  text-align: center;
+  padding-top: 32px;
+  padding-bottom: 32px;
```

Comment:
32px comes from the original empty zone, before this redesign:

```
<Box sx={{ textAlign: "center", my: 4 }}>
```

It can be adjusted, of course.